docs/user/site.rst

```diff
@@ -488,6 +493,11 @@ config_mode \: optional
               },
               zoom = 13,
               -- openlayers_url = 'http://ffac.example.org/openlayer',
+              -- tile_layer = {
+              --   type = 'XYZ',
+              --   url = 'https://{a-c}.tile.openstreetmap.org/{z}/{x}/{y}.png',
```

Comment:
Maybe we can use the preexsiting url parameter to reduce the amount of variables in the siteconf?

Reply:
@2tata What preexisting URL parameter? The `openlayers_url` is where the JS and CSS code for OpenLayers is loaded from, it has nothing to do with the layer data.

Reply:
ah, I was not entirely sure.